owner-onboarding-server/src/main.rs

```diff
@@ -141,10 +157,58 @@ async fn report_to_rendezvous(udt: OwnerServiceUDT) -> Result<()> {
     Ok(())
 }
 
+async fn check_registration_window(udt: OwnerServiceUDT) -> Result<()> {
```

Comment:
We've discussed that this got pulled out separately because the query mechanism doesn't quite do what we'd want.
I'm going to look at improving and/or documenting that, and then get back to this shortly.